enterprise/server/remote_execution/platform/platform.go

```diff
@@ -71,6 +71,7 @@ const (
 
 	RecycleRunnerPropertyName            = "recycle-runner"
 	AffinityRoutingPropertyName          = "affinity-routing"
+	PriorityPropertyName                 = "priority"
```

Comment:
can we use https://bazel.build/reference/command-line-reference#flag--remote_execution_priority instead of a platform prop for this?

Reply:
ah nice, didn't know about that.

Kind of ugly that it uses a backwards notion of priority (higher priority = executes later??) but it's probably better to stick with something that's already in the spec.

I also went ahead and limited the priorities so that they have to be between -1000 and 1000, just in case we want more flexibility later in how we interpret that field.